src/Config.cpp

```diff
@@ -263,6 +263,8 @@ parse_sloppiness(const std::string& value)
       result |= SLOPPY_LOCALE;
     } else if (token == "modules") {
       result |= SLOPPY_MODULES;
+    } else if (token == "compile_command" || token == "compile_commands") {
```

Comment:
While a ccache call creates a single compile_command file, these are then joined into a "compile_commands.json" file, so I'd leave this maybe undocumented but related name here and maybe save someone a bit of confusion.